war/pom.xml

```
@@ -322,7 +322,7 @@ THE SOFTWARE.
                 <artifactItem>
                   <groupId>org.jenkins-ci.plugins</groupId>
                   <artifactId>script-security</artifactId>
-                  <version>1172.v35f6a_0b_8207e</version>
+                  <version>1184.v85d16b_d851b_3</version>
```

1189.vb_a_b_7c8fd5fde picks up the JENKINS-69899 fix:

```suggestion
                  <version>1189.vb_a_b_7c8fd5fde</version>
```